packages/framework/src/DataFixtures/DemoMultidomain/CategoryDataFixture.php

```diff
@@ -128,6 +128,7 @@ private function editCategoryOnDomain2($referenceName, $descriptionDomain2)
         $categoryData = $this->categoryDataFacade->createFromCategory($category);
         $domainId = 2;
         $categoryData->descriptions[$domainId] = $descriptionDomain2;
+        $categoryData->enabled[$domainId] = true;
```

Comment:
Is this necessary? The create-domain-data should take care of non-nullable multidomain parameters automatically.

Reply:
Not now.